f1pystats/f1pystats.py

```diff
@@ -27,228 +27,257 @@
 
 def get_drivers(year: int, round_num: int = None):
     """Returns a list of drivers for a specified year"""
-    if round_num is None:
-        json_data = _get_json_content_from_url(f"https://ergast.com/api/f1/{year}/drivers.json")
+    if year < 1950 and year > 2022:
+        raise ValueError("Only years between 1950 and 2022 are considered as valid value for year")
     else:
-        json_data = _get_json_content_from_url(
-            f"https://ergast.com/api/f1/{year}/{round_num}/drivers.json"
+        if round_num is None:
+            json_data = _get_json_content_from_url(f"https://ergast.com/api/f1/{year}/drivers.json")
+        else:
+            json_data = _get_json_content_from_url(
+                f"https://ergast.com/api/f1/{year}/{round_num}/drivers.json"
+            )
+
+        dr_info = DriverInfo(json_data["MRData"]["DriverTable"]["Drivers"])
+
+        dr_name = dr_info.get_drivers_names()
+        dr_dob = dr_info.get_drivers_dob()
+        dr_nationality = dr_info.get_drivers_nationality()
+        dr_perm_number = dr_info.get_drivers_number()
+
+        return pd.DataFrame(
+            zip(dr_name, dr_perm_number, dr_nationality, dr_dob),
+            columns=["Drivers", "Permanent Number", "Nationality", "Date of Birth"],
         )
 
-    dr_info = DriverInfo(json_data["MRData"]["DriverTable"]["Drivers"])
-
-    dr_name = dr_info.get_drivers_names()
-    dr_dob = dr_info.get_drivers_dob()
-    dr_nationality = dr_info.get_drivers_nationality()
-    dr_perm_number = dr_info.get_drivers_number()
-
-    return pd.DataFrame(
-        zip(dr_name, dr_perm_number, dr_nationality, dr_dob),
-        columns=["Drivers", "Permanent Number", "Nationality", "Date of Birth"],
-    )
-
 
 def driver_standings(year: int):
     """Returns the driver standings for a specified year"""
-    json_data = _get_json_content_from_url(f"https://ergast.com/api/f1/{year}/driverStandings.json")
-    standings_json = json_data["MRData"]["StandingsTable"]["StandingsLists"][0]["DriverStandings"]
-    d_res = DriverResults(standings_json)
-    positions = d_res.get_driver_positions()
-    names = d_res.get_driver_names()
-    teams = d_res.get_driver_teams()
-    nationalities = d_res.get_driver_nationality()
-    points = d_res.get_driver_points()
-    wins = d_res.get_driver_wins()
-
-    return pd.DataFrame(
-        zip(positions, names, nationalities, teams, points, wins),
-        columns=["POS", "Driver", "Nationality", "Constructor", "Points", "Wins"],
-    )
+    if year < 1950 and year > 2022:
```

Comment:
This cannot be ```and```. You cannot have a year that is less than 1950 and also greater than 2022 at the same time.

It should be ```or```, instead of ```and```.